cmake/onnxruntime_unittests.cmake

```diff
@@ -708,13 +708,21 @@ endif()
 
 set(test_all_args)
 if (onnxruntime_USE_TENSORRT)
-    # TRT EP CI takes much longer time when updating to TRT 8.2
-    # So, we only run trt ep and exclude other eps to reduce CI test time.
-    #
-    # The test names of model tests were using sequential number in the past.
-    # This PR https://github.com/microsoft/onnxruntime/pull/10220 (Please see ExpandModelName function in model_tests.cc for more details)
-    # made test name contain the "ep" and "model path" information, so we can easily filter the tests using cuda ep or other ep with *cpu__* or *xxx__*.
-    list(APPEND test_all_args "--gtest_filter=-*cpu__*:*cuda__*" )
+    if (onnxruntime_SKIP_AND_PERFORM_FILTERED_TENSORRT_TESTS)
+       # TRT EP package pipelines takes much longer time to run tests with TRT 8.5. We can't use placeholder to reduce testing time due to application test deadlock. 
```

Comment:
what's the impact of this? 
how much did test time increase and what test coverage do we lose?

Reply:
can we make the timeout configurable and schedule a daily run which runs through all the tests?

Reply:
> what's the impact of this? how much did test time increase and what test coverage do we lose?
The test time is 2.5 hours for TRT 8.4 to finish, but it increases to more than 9 hours for TRT 8.5 still not even finished. (I think it needs several more hours to finish)

With this change, we won't test any unit tests instead `TensorrtExecutionProviderTest`, but we will run model tests.

Reply:
> can we make the timeout configurable and schedule a daily run which runs through all the tests?

Yes, we can. I made several commits before to add timeout configurable.